pr-data.csv

```diff
@@ -5911,16 +5911,16 @@ https://github.com/square/dagger,0fa73e0ee07015f191c25b1f3fdf7048321d1610,core,d
 https://github.com/square/keywhiz,52e77fd9f477637d5ed36ed6d1815b9c96c06166,api,keywhiz.api.model.SanitizedSecretWithGroupsListAndCursorTest.deserializesCorrectly,ID,,,
 https://github.com/square/otto,2a67589abd91879cf23a8c96542b59349485ec3d,otto,com.squareup.otto.outside.AnnotatedHandlerFinderTest.neitherOverriddenNorAnnotated,UD,RepoArchived,,
 https://github.com/square/pagerduty-incidents,c95398ffcb246082bfccddf36fcb97c877a17204,.,com.squareup.pagerduty.incidents.EventTest.serialization,ID,Opened,https://github.com/square/pagerduty-incidents/pull/30,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CancelDisposeTest.cancelDoesNotDispose,NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CompletableThrowingSafeSubscriberTest.throwingInOnCompleteDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CompletableThrowingTest.bodyThrowingInOnErrorDeliveredToPlugin,NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CompletableThrowingTest.throwingInOnCompleteDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.ObservableThrowingTest.responseThrowingInOnCompleteDeliveredToPlugin,NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.bodyThrowingInOnSuccessDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.responseThrowingInOnErrorDeliveredToPlugin,NDOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.SingleThrowingTest.responseThrowingInOnSuccessDeliveredToPlugin,NDOD;NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit,retrofit2.CallTest.cancelOkHttpRequest,NOD,,,
-https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit,retrofit2.CallTest.cancelRequest,NDOD;NOD,,,
+https://github.com/square/retrofit,ae28c3d5f6fe54f6ca409cdf82106efb47609f8f,retrofit-adapters/rxjava,retrofit2.adapter.rxjava.CancelDisposeTest.cancelDoesNotDispose,NOD,MovedToGradle,,
```

Comment:
As above -- add commits when the project moved to Gradle. Did you try to run this project with the NonDex Gradle plugin?